src/docfx/template/JsonSchemaProvider.cs

```diff
@@ -75,6 +75,17 @@ public RenderType GetRenderType(ContentType contentType, SourceInfo<string?> mim
             };
         }
 
+        public bool IsContentRenderType(string? mime)
```

Comment:
Why not using `GetRenderType`, this method seems like a duplication of `GetRenderType`.

Reply:
There's no context to get `ContentType` here, the only input left is `schema`